syntax="proto3";

package sdk_curve;

enum CurveCalMethod
{
    NONE = 0;
    ADD = 1;
    AVG = 2;
    MAX = 3;
    MIN = 4;
    MIDDLE = 5;
}

message CurveCal
{
	CurveCalMethod method = 1;
	double value = 2;
}

message CurveData
{
    string app_mark=1;                                         
    map<string, string> tag_set=2;                    
    map<string, CurveCal> metric_val=3;      
    string instance_mark=4;
}
message SDKCurvePkg
{
    repeated CurveData curve_datas=1;                                
    int64 seq=2;
    string version=3;

}
